java/src/main/java/com/company/automation/fundamentals/optional_task/optional_task_01/OptionalTask01.java

```diff
@@ -0,0 +1,414 @@
+package com.company.automation.fundamentals.optional_task.optional_task_01;
+
+/*
+Задание. Ввести n чисел с консоли.
+
+1. + Найти самое короткое и самое длинное число. Вывести найденные числа и их длину.
+2. + Вывести числа в порядке возрастания (убывания) значений их длины.
+3. + Вывести на консоль те числа, длина которых меньше (больше) средней длины по всем числам, а также длину.
+4. + Найти число, в котором количество различных цифр минимально. Если таких чисел несколько, найти первое из них.
+5. + Найти количество чисел, содержащих только четные цифры, а среди оставшихся — количество чисел с равным числом
+     четных и нечетных цифр.
+6. + Найти число, цифры в котором идут в строгом порядке возрастания. Если таких чисел несколько, найти первое из них.
+7. + Найти число, состоящее только из различных цифр. Если таких чисел несколько, найти первое из них.
+ */
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class OptionalTask01 {
+    public static void main(String[] args) {
+        String numbersLine;
+        String[] stringOfNumbers;
+
+        numbersLine = inputIntegersAsAString();
+
+        stringOfNumbers = numbersLine.split("\\s+");
```

Comment:
Please mix the declaration and initialization in one line.

Reply:
Fixed